lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -154,9 +154,18 @@ Parser.prototype.onFrameBody = function onFrameBody(header, buffer, callback) {
   else if (header.type === frameType.X_FORWARDED_FOR)
     this.onXForwardedFrame(header, buffer, callback);
   else
-    callback(null, { type: 'unknown: ' + header.type });
+    this.onUnknownFrame(header, buffer, callback);
 };
 
+Parser.prototype.onUnknownFrame = function onUnknownFrame(header, buffer, callback) {
```

Comment:
Argh, anyway. Let's do it the way you do it :) I don't care that much about it. We can always figure it out later.